### ISSUE TITLE
How to fix Class "Doctrine\Bundle\FixturesBundle\Fixture" not found while loading "Wallabag\AnnotationBundle\DataFixtures\AnnotationFixtures"

### ISSUE THREAD

**Issue by jason-weiser** [wallabag/wallabag#6651]
### Environment

* **Version**: 2.5.4 > 2.6.0
* **Installation**: git clone and then make install
* **PHP version**: 8.1.2
* **OS**: Debian 11
* **Database**: MariaDB
* **Parameters**:

My app/config/parameters.yml is:

  ```
    database_driver: pdo_mysql
    database_host: 127.0.0.1
    database_port: 3306
    database_name: wallabag
    database_user: wallabaguser
    database_password: notactuallypassword
    database_path: null
    database_table_prefix: wallabag_
    database_socket: null
    database_charset: utf8mb4
    domain_name: 'https://wallabag.something.tld
    locale: en
    secret: supersecretthing
    twofactor_auth: true
    twofactor_sender: no-reply@wallabag.org
    fosuser_registration: true
    fosuser_confirmation: true
    from_email: no-reply@wallabag.org
    rss_limit: 50
    rabbitmq_host: localhost
    rabbitmq_port: 5672
    rabbitmq_user: guest
    rabbitmq_password: guest
    rabbitmq_prefetch_count: 10
    redis_scheme: tcp
    redis_host: localhost
    redis_port: 6379
    redis_path: null
    redis_password: null
    server_name: wallabag.something.tld
    fos_oauth_server_access_token_lifetime: 3600
    fos_oauth_server_refresh_token_lifetime: 1209600
    sentry_dsn: null
    mailer_dsn: 'smtp://127.0.0.1'
  
  ```

### What steps will reproduce the bug?

update the instance with `make update`

Goes fine until, at the end, it reads:

```
In DefinitionErrorExceptionPass.php line 54:

Class "Doctrine\Bundle\FixturesBundle\Fixture" not found while loading "Wallabag\AnnotationBundle\DataFixtures\AnnotationFixtures".

Script bin/console cache:clear --no-warmup handling the post-cmd event returned with error code 1
Script @post-cmd was called via post-install-cmd
```

My Wallabag instance is inaccessible, now, and I'm not sure how to fix it. 

This issue was reportedly fixed with #6648 but I'm not sure how to un-break my own install.

Thanks!

EDIT: It also does this with a fresh install where I run:

```
git clone https://github.com/wallabag/wallabag.git
cd wallabag && make install
```

and use the parameters file detailed above.

**Comment by nicosomb** [comment]
@jason-weiser can you try a new `make update` please? 

We released 2.6.1. Before publishing the blog post, I want to have news from you with this issue.

**Comment by nicosomb** [comment]
This bug is fixed thanks to 2.6.1.

**Comment by jason-weiser** [comment]
> This bug is fixed thanks to 2.6.1.

Sorry for the delay, yep! 2.6.1 fixed the issue. Everything is working again.

Wallabag is awesome, btw. It was one of the first things I selfhosted and it has been going strong for me for years. Thanks so much for the work you and the team do on it.
